src/pages/index.js

```diff
@@ -0,0 +1,305 @@
+import "./index.css";
+import Api from "../utils/Api.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "1810d547-91a3-4cf2-9027-84faa0c5dd2e",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([cards, users]) => {
+    cards.forEach((card) => {
+      const cardEl = getCardElement(card);
+      renderCard(card);
+    });
+
+    editProfileName.textContent = users.name;
+    editProfileDescription.textContent = users.about;
+    avatarImage.src = users.avatar;
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+import { enableValidation } from "../scripts/validation.js";
+import { disableButton } from "../scripts/validation.js";
+import { settings } from "../scripts/validation.js";
+import { resetValidation } from "../scripts/validation.js";
+
+import logoSrc from "../images/logo.svg";
+import avatarSrc from "../images/avatar.jpg";
+import pencilSrc from "../images/pencil.svg";
+import newPostSrc from "../images/new_post.svg";
+import pencilAvatarSrc from "../images/pencil-white.svg";
+
+const logoImage = document.getElementById("logo-svg");
+logoImage.src = logoSrc;
+const avatarImage = document.getElementById("avatar");
+avatarImage.src = avatarSrc;
+const pencilImage = document.getElementById("pencil-svg");
+pencilImage.src = pencilSrc;
+const newPostImage = document.getElementById("new_post-svg");
+newPostImage.src = newPostSrc;
+const avatarEdit = document.getElementById("pencil-avatar");
+avatarEdit.src = pencilAvatarSrc;
+
+import galaxyImage from "../images/1.2scott-lord-hnBcyFZR--Q-unsplash.jpg";
+import pacificNorthWestImage from "../images/2.2aiden-_luqEchlLxQ-unsplash.jpg";
+import fallImage from "../images/3.2noah-silliman-y3IwQ9hUE4A-unsplash.jpg";
+import rockyMountainsImage from "../images/4.2mike-scheid-xoYPV4oVQJI-unsplash.jpg";
+import gravelBikeImage from "../images/5.2alessio-soggetti-sJE9qSemdFk-unsplash.jpg";
+import pikesPlaceImage from "../images/6.2david-izquierdo-XXec738eA9I-unsplash.jpg";
```

Comment:
Please, delete the unused imports not to mislead webpack and the other developers